[BUG] ./magefile.go:14:11: undefined: mageutil.Build Error: error compiling magefiles

### OpenIM Server Version

3.7

### Operating System and CPU Architecture

Linux (AMD)

### Deployment Method

Docker Deployment

### Bug Description and Steps to Reproduce

./magefile.go:14:11: undefined: mageutil.Build
Error: error compiling magefiles


### Screenshots Link

![3a5246b52bc9eae20369bfa34576a02](https://github.com/openimsdk/open-im-server/assets/72860476/f5fe4b08-51ef-47ae-b4b5-8b49c8046aa4)

--- a/Dockerfile
+++ b/Dockerfile
@@ -43,7 +43,7 @@ COPY --from=builder $SERVER_DIR/start-config.yml $SERVER_DIR/
 COPY --from=builder $SERVER_DIR/go.mod $SERVER_DIR/
 COPY --from=builder $SERVER_DIR/go.sum $SERVER_DIR/
 
-RUN go get github.com/openimsdk/gomake@v0.0.9
+RUN go get github.com/openimsdk/gomake@v0.0.13
 
 # Set the command to run when the container starts
 ENTRYPOINT ["sh", "-c", "mage start && tail -f /dev/null"]

